web/src/components/Callout.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactElement } from "react";
+
+export type CalloutTypes = "informational" | "conditional" | "warning" | "note";
+
+interface Props {
+  calloutType: CalloutTypes;
+  header?: string;
+  children: string;
+  icon?: "true" | "false";
+}
+
+interface CalloutMappingObj {
+  containerStyling: string;
+  headingStyling: string;
+  headingDefaultText: string;
+  bodyStyling: string;
+  iconStyling: string;
+}
+
+export const calloutComponentMapping: Record<string, CalloutMappingObj> = {
+  informational: {
+    containerStyling: "bg-accent-cool-lightest",
+    headingStyling: "text-accent-cool-more-dark",
+    headingDefaultText: "Keep in mind:",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-informational-icon",
+  },
+  conditional: {
+    containerStyling: "bg-primary-extra-light",
+    headingStyling: "text-primary-darker",
+    headingDefaultText: "Once completed, you will have...",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-conditional-icon",
+  },
+  warning: {
+    containerStyling: "bg-warning-extra-light",
+    headingStyling: "text-accent-warm-darker",
+    headingDefaultText: "Be advised:",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-warning-icon",
+  },
+  note: {
+    containerStyling: "bg-base-lightest",
+    headingStyling: "text-base-darkest",
+    headingDefaultText: "Note:",
+    bodyStyling: "text-primary-darker",
+    iconStyling: "callout-note-icon",
+  },
+};
+
+export const Callout = (props: Props): ReactElement => {
+  return (
+    <div
+      className={`padding-205 radius-md margin-bottom-2 ${
+        calloutComponentMapping[props.calloutType].containerStyling
+      }`}
+    >
+      <div className="flex">
+        {props.icon === "true" && (
```

Comment:
[pebble] I don't like that we doing a string comparison here. Let's turn this into a boolean. Then the check can just be:
```suggestion
        {props.icon && (
```